packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -133,27 +120,16 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
             <SendBox {...sendBoxProps} styles={sendBoxStyles} />
           </Stack.Item>
         </Stack>
-        {options?.participantPane === true && (
-          <Stack className={participantListWrapper}>
-            <Stack className={participantListStack}>
-              <Stack.Item className={listHeader}>{chatListHeader}</Stack.Item>
-              <FocusZone className={participantListStyle}>
-                <ParticipantList
-                  {...participantListProps}
-                  onRenderAvatar={(userId, options) => (
-                    <AvatarPersona
-                      data-ui-id="chat-composite-participant-custom-avatar"
-                      userId={userId}
-                      {...options}
-                      dataProvider={onFetchAvatarPersonaData}
-                    />
-                  )}
-                  onFetchParticipantMenuItems={onFetchParticipantMenuItems}
-                />
-              </FocusZone>
-            </Stack>
-          </Stack>
-        )}
+        {
+          /* beta:start */
```

Comment:
Extract beta component logic into a single file is the best way to minimize beta blocks - which helps us make smaller files and clearer logic